.github/workflows/ci.yaml

```diff
@@ -36,6 +36,7 @@ jobs:
     env:
       MYPY_FORCE_COLOR: 1
       TERM: xterm-color
+      SETUPTOOLS_USE_DISTUTILS: stdlib
```

Comment:
What does this line do? Just for my knowledge.

Reply:
@sunank200 I believe this comment explains the key issue:
https://github.com/pypa/setuptools/issues/2965#issuecomment-1002178847

```
The way the distutils hack works, it makes Setuptools' own copy of distutils take precedence over the standard library, so causes import distutils to put sys.modules['distutils'] = <setuptools._distutils>. So it's not a big surprise to me that running the tests from the standard library against code not in the standard library will have problems.
```

More information in the official docs:
https://setuptools.pypa.io/en/latest/deprecated/distutils-legacy.html